pyface/resource/resource_manager.py

```diff
@@ -113,14 +115,14 @@ def _locate_image(self, image_name, resource_path, size):
             # If we come across a reference to a module, use pkg_resources
             # to try and find the image inside of an .egg, .zip, etc.
             if isinstance(dirname, types.ModuleType):
-                from pkg_resources import resource_string
-
                 for path in subdirs:
                     for extension in extensions:
                         searchpath = "%s/%s%s" % (path, basename, extension)
                         try:
-                            data = resource_string(
-                                dirname.__name__, searchpath
+                            data = (
+                                files(dirname.__name__)
```

Comment:
I am not sure about this... how does this work? I think `files` returns an iterable? Documentation says it can return None too: https://docs.python.org/3/library/importlib.metadata.html

Reply:
this is the recommended migration as documented here - https://importlib-resources.readthedocs.io/en/latest/migration.html#pkg-resources-resource-string.

Reply:
I was reading `importlib_metadata` documentation but this comes from `importlib_resources`. 🤦 
Thanks.